src/styles/blocks/star.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: 50%;
+
+    :not(:last-child) {
+      margin-right: 4px;
+    }
+
+  }
+  :nth-child(-n+4) {
```

Comment:
add selectors for all cases (1, 2 stars, etc):
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/fea14420-0d16-4694-88d9-8a9204bf90f2)